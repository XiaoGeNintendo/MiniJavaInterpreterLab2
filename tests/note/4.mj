class A{

}

class B{
    string to_string(){
        return "hello";
    }
}

class C extends B{
}

int main(){
    A a=new A();
    B b=new B();
    C c=new C();
    C npt;
    println(a);
    println(b);
    println(c);
    println(npt);
    print(a);
    print(b);
    print(c);
    print(npt);
    return 0;
}
