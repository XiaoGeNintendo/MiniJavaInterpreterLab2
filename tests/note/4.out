A
hello
hello
null
AhellohellonullProcess exits with the code 0.
